pkg/provision/sync/sync.go

```diff
@@ -200,17 +200,19 @@ func printDiff(specObj, clusterObj crclient.Object, log logr.Logger) {
 			diffOpts = podDiffOpts
 		case *corev1.ConfigMap:
 			diffOpts = configmapDiffOpts
-		case *corev1.Secret:
-			diffOpts = secretDiffOpts
 		case *v1alpha1.DevWorkspaceRouting:
 			diffOpts = routingDiffOpts
 		case *networkingv1.Ingress:
 			diffOpts = ingressDiffOpts
 		case *routev1.Route:
 			diffOpts = routeDiffOpts
+		case *corev1.Secret:
+			log.Info(fmt.Sprintf("Diff: secret %s data upated", specObj.GetName()))
```

Comment:
Why are we no longer printing the secret diff? To reduce logging? Or to make it more obvious that it's a secret that was updated?